modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryReaderEx.java

```diff
@@ -46,13 +46,6 @@ public interface BinaryReaderEx extends BinaryReader, BinaryRawReader, BinaryRea
      */
     public BinaryInputStream in();
 
-    /**
-     * Get or create object schema.
-     *
-     * @return Schema.
-     */
-    public BinarySchema getOrCreateSchema();
```

Comment:
Why do you remove this method? Reader is an internal interface too.

Reply:
`BinaryReaderEx` and `BinaryWriterEx` are currently used outside of `internal.binary` package. It looks strange if package-private class `BinarySchema` will be exposed outside.